components/setting_picture.jsx

```diff
@@ -143,6 +144,25 @@ export default class SettingPicture extends Component {
                     src={this.props.src}
                 />
             );
+
+            if (this.props.onRemove) {
+                img = (
+                    <div className={`${imageContext}-img__container`}>
+                        <img
+                            ref='image'
```

Comment:
Do we actually use this `ref` anywhere?

Reply:
Just realized not, I removed it now. Thanks!